resource_customizations/platform.confluent.io/Connect/health.lua

```diff
@@ -0,0 +1,19 @@
+hs = {}
+if obj.status ~= nil then
+  if obj.status.phase ~= nil then
+    if obj.status.phase == "RUNNING" then
+      hs.status = "Healthy"
+      hs.message = "Connect running"
+      return hs
+    end
+    if obj.status.phase == "PROVISIONING" then
+      hs.status = "Progressing"
+      hs.message = "Connect provisioning"
+      return hs
+    end
+  end
+end
+
+hs.status = "Unknown"
```

Comment:
We never return `Unknown` health state in checks, but instead use `Progressing`

Reply:
Thanks, Fixed.